scripts/split-e2e-tests.ts

```diff
@@ -118,14 +110,34 @@ const RUN_SOLO: (string | RegExp)[] = [
   'src/__tests__/transformer-migrations/model-migration.test.ts',
   'src/__tests__/graphql-v2/searchable-node-to-node-encryption/searchable-previous-deployment-no-node-to-node.test.ts',
   'src/__tests__/graphql-v2/searchable-node-to-node-encryption/searchable-previous-deployment-had-node-to-node.test.ts',
+  /src\/__tests__\/api_1.*\.test\.ts/,
   // GraphQL E2E tests
   'src/__tests__/FunctionTransformerTestsV2.e2e.test.ts',
   'src/__tests__/HttpTransformer.e2e.test.ts',
   'src/__tests__/HttpTransformerV2.e2e.test.ts',
   // Deploy Velocity tests
   /src\/__tests__\/deploy-velocity\/.*\.test\.ts/,
-  // RDS tests
+  // SQL tests
+  /src\/__tests__\/rds-.*\.test\.ts/,
+  // CDK tests
+  /src\/__tests__\/base-cdk.*\.test\.ts/,
+  'src/__tests__/amplify-table-1.test.ts',
+  'src/__tests__/api_canary.test.ts',
+  'src/__tests__/sql-models.test.ts',
+];
+
+const RUN_IN_ALL_REGIONS = [
+  // DDB tests
+  'src/__tests__/api_canary.test.ts',
+  // CDK tests
+  'src/__tests__/base-cdk.test.ts',
+];
+
+const RUN_IN_NON_OPT_IN_REGIONS: (string | RegExp)[] = [
+  // SQL tests
   /src\/__tests__\/rds-.*\.test\.ts/,
+  // Searchable tests
+  /src\/__tests__\/.*searchable.*\.test\.ts/,
```

Comment:
We have items in our backlog like [Manifest for the regionalized SNS topics](https://app.asana.com/0/1206048977824382/1206213458097061/f), [Add me-south-1 to beta stage](https://app.asana.com/0/1206048977824382/1206648775264573/f) and [Searchable support for eu-south-1](https://app.asana.com/0/1206048977824382/1206648775264575/f) that need to be addressed before these can be supported.